cypress/integration/page.spec.js

```diff
@@ -489,7 +489,7 @@ describe('', () => {
       });
 
       it('should show a temp TodoItem with correct title', () => {
-        todos.assertTitle(5, 'Test Todo');
+        todos.assertTitle(5, 'Test Todo.tsx');
```

Comment:
Don't need to change tests, fix it everywhere
```suggestion
        todos.assertTitle(5, 'Test Todo');
```